server/src/messages/messages.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@nestjs/common';
+import { Message } from './entities/message.entity';
+import { User } from '../users/entities/user.entity';
+import { Repository } from 'typeorm';
+import { CreateMessageDto } from './dto/create-message.dto';
+import { UpdateMessageDto } from './dto/update-message.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+
+@Injectable()
+export class MessagesService {
+  constructor(
+    @InjectRepository(Message) private messagesRepository: Repository<Message>,
+  ) {}
+
+  async create(createMessageDto: CreateMessageDto) {
```

Comment:
add return types if you can
`async create(createMessageDto: CreateMessageDto): Message[] {`
or something like that
i think i went over this in the recorded PR review of keith's Asset class
lmk if you need a link to the video

Reply:
also, does this one need to be `async`?
should the others be `async` as well?
(for async functions return type would be `Promise<something>`)

Reply:
Having had a look at Keith's Assets code, I'm assuming `Promise<Message[]>` would the syntax when you're expecting an array of messages back, and `Promise<Message>` would be the syntax for just one?